lp-steam/templates/csgo/server.j2

```diff
@@ -22,14 +22,34 @@ sv_lan 1
 host_players_show 2
 host_info_show 2
 
+game_type 0 
```

Comment:
`game_type` and `game_mode` are set from the command line, setting them here will do little to protect them from being changed. I propose setting this in a private exec config if you want to be sure.